sbtc-signer-prd.md

```diff
@@ -0,0 +1,100 @@
+# sBTC Signer | Product Requirements Document
+
+>Detailed Technical Specifications Document can be found [here](sbtc-tsd.md).
+
+>Detailed Product Requirements Document for sBTC can be found [here](sbtc-prd.md).
+
+# Introduction
+
+This document outlines the requirements for the development of the sBTC Signer, providing guidance and allowing for potential refinements based on continuous feedback and project progress. The requirements are derived from the Signer user stories, which serve as a foundational reference.
+
+## Requirement 1: Integration with Transaction Monitoring API
+
+**Description:**
+
+The system must enable seamless integration with the Chainalysis API to facilitate blockchain transaction analysis, enhancing security and compliance. The product will offer the necessary tools and configuration options to facilitate API calls by third-party applications.
+
+**Requirements:**
+
+| No. | Task |
+| --- | --- |
+| 1 | Support third-party applications to make Chainalysis API calls. |
+| 2 | Provide minimal configuration control for secure and flexible API calls, including a list of auto-deny addresses for flagged transactions. |
+| 3 | Implement auto-deny capability for transactions based on Chainalysis API analysis results. |
+
+## Requirement 2: Trigger Manual Review for Transactions Above Certain Threshold
+
+**Description:**
+
+The system should automatically initiate a manual review process for specific transactions based on predefined conditions to ensure compliance and prevent suspicious activities.
+
+**Requirements:**
```

Comment:
would this serve better as a checkbox, or just a list? a table doesn't seem right here